quantmsio/core/quantms/msstats_in.py

```
@@ -0,0 +1,702 @@
+from quantmsio.utils.pride_utils import clean_peptidoform_sequence
+
+
+class MsstatsIN(DuckDB):
+    def __init__(
+        self,
+        report_path: Union[Path, str],
+        sdrf_path: Union[Path, str],
+        duckdb_max_memory="16GB",
+        duckdb_threads=4,
+    ):
+        super(MsstatsIN, self).__init__(report_path, duckdb_max_memory, duckdb_threads)
```

Pylint (3.3.7)

[refactor] Consider using Python 3 style super() without arguments

(R1725)